src/DynamoCore/Models/DynamoModel.cs

```diff
@@ -749,6 +775,11 @@ protected DynamoModel(IStartConfiguration config)
             StartBackupFilesTimer();
 
             TraceReconciliationProcessor = this;
+            //we're done loading the model
```

Comment:
This comment is self-evident. Maybe you want to write something like `This event should only be raised when at the end of this method`.